requirements.d/development.txt

```diff
@@ -11,3 +11,4 @@ pytest-benchmark
 Cython!=0.27
 twine
 python-dateutil
+pyinstaller
```

Comment:
I'ld rather not have that there:

- for normal development, it is not needed
- for normal testing, it is also not needed
- for building the linux/freebsd/macOS binaries, I use a pyinstaller which I build from git sources after  checking out my git repo ( https://github.com/ThomasWaldmann/pyinstaller/commits/v4.7-maint ), which i maintain with important patches, so I don't want another pyinstaller in the PATH.

Reply:
I can create a development-win.txt which contains only this addition, but I would have no redundancy in such config files.
I'd rather do it as you like me to do.

Reply:
I'ld not put it into development*.txt (not for any OS) and also not create a duplicate file we also need to update with changes.

pyinstaller is not needed for general development work, but only if one wants to build the binary.

And, as I already pointed out, if you want to build the binary, you maybe want to use v4.7-maint branch pyinstaller code with maintenance fixes.

Reply:
https://github.com/borgbackup/borg/blob/master/tox.ini#L19

also, tox installs all that stuff into each testenv - so having pyinstaller in there would significantly slow down test setup for each python version, locally, in vagrant, on github, ...

Reply:
I removed it again.